docs/docs/cloud/known_issues.md

```diff
@@ -28,7 +28,7 @@ See related issue: [#6853](https://github.com/meltano/meltano/issues/6853).
 Meltano cloud will have support for automatic deployments in GA but currently users need to manually request a re-deployment of a project when new commits are added to the git repository.
 
 ```bash
-meltano-cloud deployment update --name prod
```

Comment:
Definitely don't want to drop the `-` just yet

Reply:
We were using `meltano cloud` rather than `meltano-cloud` in several places throughout these docs. I went ahead and standardized to `meltano-cloud` rather than `meltano cloud` for the time being.